packages/valory/skills/decision_maker_abci/behaviours/base.py

```diff
@@ -618,6 +624,9 @@ def _write_benchmark_results(
             )
             return
 
+        if liquidity_info is None:
+            self.context.logger.info("No market liquidity information.")
```

Comment:
Here you can instead check if all the attributes are `None`. You can create a method in the dataclass for that.